arkane/ess/gaussian.py

```diff
@@ -378,6 +378,11 @@ def load_scan_energies(self):
                 # to the optimized geometry
                 if 'Optimization completed' in line:
                     vlist.append(energy)
+                # In some cases, the optimization cannot converge within the given steps.
+                # Then, the geometry is not optimized. we need to exclude these values.
+                if 'Optimization stopped' in line:
+                    non_optimized.append(len(vlist))
+                    vlist.append(energy)
```

Comment:
Does `non_optimized` ever get initialized anywhere? I think this is the reason why the Travis tests are failing, unless I am missing something obvious.

Reply:
Yeah, you are definitely correct! Thanks, Mark.